ospfd/ospf_vty.c

```diff
@@ -12108,7 +12108,7 @@ static int config_write_ospf_external_aggregator(struct vty *vty,
 		if (rn->info) {
 			struct ospf_external_aggr_rt *aggr = rn->info;
 
-			vty_out(vty, " summary-address %pI4/%d ",
+			vty_out(vty, " summary-address %pI4/%d",
 				&aggr->p.prefix, aggr->p.prefixlen);
 			if (aggr->tag)
 				vty_out(vty, " tag %u ", aggr->tag);
```

Comment:
Can you remove the extra spaces here at the end as well, the same problem as above exists here too.

Reply:
Yes, thanks.
I have modified it and re-pushed it.